src/lib/tsc.ts

```diff
@@ -133,8 +154,7 @@ export function registerTypeScriptTranspile(compilerOptions?: ts.CompilerOptions
       })
 
       if (transpiled.diagnostics && transpiled.diagnostics.length > 0) {
-        console.log(ts.formatDiagnosticsWithColorAndContext(transpiled.diagnostics, diagnosticHost))
```

Comment:
No need to console.log as the `process.on('unhandledError', ...)` catches it and log it already